components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -179,12 +179,17 @@ def changeActiveGroup(self, gid):  # TODO: should be moved to ISession
             return False
 
     def getOrphanedContainerSettings(self):
-        name = (self.getConfigService().getConfigValue(
-                "omero.client.ui.tree.orphans.name") or "Orphaned image")
-        description = (self.getConfigService().getConfigValue(
-                       "omero.client.ui.tree.orphans.description") or
-                       "This is a virtual container with orphaned images.")
-        return name, description
+        orphans = dict()
+        if toBoolean(self.getConfigService().getConfigValue(
+                "omero.client.ui.tree.orphans.enabled")):
+            orphans['name'] = \
+                self.getConfigService().getConfigValue(
+                    "omero.client.ui.tree.orphans.name") or "Orphaned image"
+            orphans['description'] = \
+                (self.getConfigService().getConfigValue(
+                 "omero.client.ui.tree.orphans.description") or
+                 "This is a virtual container with orphaned images.")
+        return orphans
```

Comment:
When we're accessing config service in Blitz Gateway we now do try/except (see https://github.com/openmicroscopy/openmicroscopy/pull/4191) so it would be nice to do it here too, so we don't risk screwing up the session if this fails.